src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItem.cshtml

```diff
@@ -20,4 +21,4 @@
     }
 }
 
-@tag
+@tag
```

Comment:
Don't remove to EOL at the end of the files

Reply:
Put it back in.